app/cdash/include/ctestparser.php

```diff
@@ -170,8 +171,8 @@ function ctest_parse($filehandle, string $filename, $projectid, $expected_md5 =
         $xml_info['xml_handler'] = null;
         $xml_info['xml_type'] = '';
         $message = "Could not determine submission file type for: '{$filename}'";
-        Log::warning($message);
-        if ((bool) config('cdash.validate_xml_submissions') === true) {
+        Log::info($message);
+        if (config('cdash.validate_submissions') === SubmissionValidationType::REJECT) {
```

Comment:
What do you think about just failing all the time here, similar to the other lines I commented on?  Iirc, these lines were added in response to the CMake CI failures, but it's possibly ok to remove them now that we're making a major version release.  Anyone submitting junk to CDash should probably be notified...  I could see a good argument either way here though.

Reply:
The failing all the time on this check would make sense since we would fail the same point during submission.  As this is fairly removed from the user, I think we can abandon processing without giving it much thought